build/l10n-plugin.mts

```diff
@@ -78,8 +76,9 @@ export default (dir: string) => {
 				return null
 			} else if (source.endsWith('l10n.ts') && importer && !importer.includes('node_modules')) {
 				if (dirname(resolve(dirname(importer), source)).split('/').at(-1) === 'src') {
+					const [path] = importer.split('?', 2)
 					// return our wrapper for handling the import
-					return `\0l10nwrapper?source=${encodeURIComponent(importer)}`
+					return `\0l10nwrapper?source=${encodeURIComponent(path)}`
```

Comment:
For myself: change 1? to remove query?

Reply:
> Yes, this is the actual fix. Other changes are refactorings/comments

Yes sorry for that but it took me again too long to understand the plugin :sweat_smile: 
So prevent the next person to have the same problem by adding comments.